addons/binding/org.openhab.binding.hdpowerview/ESH-INF/thing/thing-types.xml

```diff
@@ -9,9 +9,10 @@
         <description>Hunter Douglas PowerView Hub</description>
         
         <config-description>
-            <parameter name="ipAddress" type="text" required="true">
+            <parameter name="ipAddress" type="text" required="true" pattern="^\d{1,3}\.\d{1,3}\.\d{1,3}$">
```

Comment:
Isn't that a bit too restrictive? It does not allow to enter a hostname instead and what about IPv6 addresses?